### ISSUE TITLE
Issue when adding tag(s) to multiple attributes (batch tagging)

### ISSUE THREAD

**Issue by Vesiix** [MISP/MISP#4088]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug, support
| OS version (server)       | Ubuntu, RedHat
| OS version (client)       | Windows 7/10
| PHP version               | 7.1 (RH), 7.2.10 Ubuntu
| MISP version / git hash   | 2.4.102
| Browser                   | Chrome

### Expected behavior
When adding a tag(s) to multiple attributes, attach the tag(s) to each attribute.

### Actual behavior
Adds tag(s) to the first attribute and states "X tags could not be added. 1 Tag(s) was added" in a red pop-up. This is specifically for everything that is not `Site Admin`.

### Steps to reproduce the behavior
1. Create an event as anything other than `Site Admin`
2. Batch import multiple attributes
3. Click "All attribute" check box
4. Click "Tag selected Attributes"
5. Select a tag
6. Click "Submit"

### Logs, screenshots, configuration dump, ...
Debug Log: 

```
2019-02-04 17:19:55 Notice: Notice (8): Undefined index: Event in [/var/www/MISP/app/Controller/AttributesController.php, line 3064]
Trace:
ErrorHandler::handleError() - APP/Lib/cakephp/lib/Cake/Error/ErrorHandler.php, line 230
AttributesController::addTag() - APP/Controller/AttributesController.php, line 3064
ReflectionMethod::invokeArgs() - [internal], line ??
Controller::invokeAction() - APP/Lib/cakephp/lib/Cake/Controller/Controller.php, line 491
Dispatcher::_invoke() - APP/Lib/cakephp/lib/Cake/Routing/Dispatcher.php, line 193
Dispatcher::dispatch() - APP/Lib/cakephp/lib/Cake/Routing/Dispatcher.php, line 167
[main] - APP/webroot/index.php, line 92
```

**Comment by Vesiix** [comment]
I tried reinstalling the AttributesController.php, TagsController.php, EventsController.php and resetting server permissions, but had no success. Also, batch tagging works for admin but nothing else, Is this a known issue? @iglocska

**Comment by Vesiix** [comment]
Error log is saying:
```
2019-02-11 06:57:33 Error: [MissingViewException] View file "Events/ajax/event_lock.ctp" is missing.
Exception Attributes: array (
  'file' => 'Events/ajax/event_lock.ctp',
)
Request URL: /events/checkLocks/4848
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(470): View->_getViewFileName('Events/ajax/eve...')
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(963): View->render('/Events/ajax/ev...', NULL)
#2 /var/www/MISP/app/Controller/EventsController.php(5226): Controller->render('/Events/ajax/ev...')
#3 [internal function]: EventsController->checkLocks('4848')
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(491): ReflectionMethod->invokeArgs(Object(EventsController), Array)
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(EventsController), Object(CakeRequest))
#7 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
```

**Comment by SteveClement** [comment]
Is this still an issue?

**Comment by Vesiix** [comment]
Yes, resolved in v2.4.106/107. Thanks!
